wurst/systems/commands/SpawningTestHelpers.wurst

```diff
@@ -46,3 +65,23 @@ init
                                 break
                     else
                         simError(triggerPlayer, "Cannot spawn {0}".format(idStr))
+
+            registerCommandAll("spawn") (triggerPlayer, args) ->
+                let troll = triggerPlayer.getTroll()
+                var pos = troll.getPos()
+                var count = args.get(2).toInt()
+                let listName = args.get(1).toLowerCase()
+                let toSpawn = itemMap.get(listName)
+                
+                if listName == null or toSpawn == null
```

Comment:
Does `itemMap.get(null)` work? It would be problematic if it didn't, as you don't check that `listName` is not `null` until after the call.

Reply:
Well `itemMap.get(null)` returns null so I figured it would be ok this way.

Reply:
OK, just wanted to make sure there wasn't error output being displayed.